grpcx/server.go

```diff
@@ -53,3 +53,26 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 		return resp, err
 	}
 }
+
+// WithErrorLogs returns a gRPC interceptor for unary calls that instrument requests
+// with logs for the errors.
+func WithErrorLogs(l logx.Logger) grpc.UnaryServerInterceptor {
+	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+		resp, err := handler(ctx, req)
+
+		if err != nil {
+			reqMsg, _ := json.Marshal(req)
+			respMsg, _ := json.Marshal(resp)
+			fields := []logx.Field{
+				{Key: "url", Value: info.FullMethod},
+				{Key: "ctx_full_method", Value: info.FullMethod},
```

Comment:
remove this line, "url" key is fine

Reply:
As commented on the channel, I'm going to log on some redundant fields, for the shake of visibility on multiple DD panels.